cairo/crates/contracts/src/libs/aggregation_ism_metadata.cairo

```diff
@@ -102,21 +92,55 @@ mod test {
                 0x00000000000000000000000000000000,
             ],
         );
+        let mut expected_result = array![0xAAAAAAAAAAAAAAAABBBBCCCC00000000_u256];
+        let mut cur_idx = 0;
+        while (cur_idx != 1) {
+            let result = AggregationIsmMetadata::metadata_at(encoded_metadata.clone(), 0);
+            assert(
+                *BytesTrait::data(result.clone())[0] == *expected_result.at(cur_idx).low,
+                'Agg metadata extract failed',
+            );
+            cur_idx += 1;
+        };
+    }
+
+    #[test]
+    fn test_metadata_not_padded() {
```

Comment:
There are a bunch of constants in this test, consider adding some comments about what and how this test works

Reply:
done